cloud/filestore/tests/python/lib/daemon_config.py

```diff
@@ -412,6 +425,17 @@ def __init__(
             ic_port=ic_port
         )
 
+        self.__diag_config=self.__generate_diag_txt()
+        with open(self.diag_config_file_path, "w") as config_file:
+            if self.__diag_config:
+                config_file.write(MessageToString(self.__diag_config))
+                os.fsync(config_file)
+
+    def __generate_diag_txt(self):
+        diag = TDiagnosticsConfig()
+        diag.ProfileLogTimeThreshold = 100
```

Comment:
if you set ProfileLogTimeThreshold to 100 msec, why do you need the Flush changes in the c++ code. Just make sure the test waits extra cycle or two before collecting the logs.
I suggest to discard all c++ changes and just make sure that [sync](https://linux.die.net/man/8/sync) is called at the end of the test and that you wait extra time (ProfileLogTimeThreshold * 2)